src/main/config/config_streamer.c

```diff
@@ -26,12 +26,41 @@
 extern uint8_t __config_start;   // configured via linker script when building binaries.
 extern uint8_t __config_end;
 
-uint32_t FlashPageSize;
+#if !defined(FLASH_PAGE_SIZE)
```

Comment:
these changes are from a seperate commit, which will be pulled into master soon, then i will rebase this branch on master